src/dataworker/PoolRebalanceUtils.ts

```diff
@@ -144,14 +144,13 @@ export function generateMarkdownForRootBundle(
     delete leaf.leafId;
     leaf.amountToReturn = convertFromWei(
       leaf.amountToReturn,
-      hubPoolClient.getTokenInfo(leaf.chainId, leaf.l2TokenAddress).decimals
+      hubPoolClient.getTokenInfoForAddress(leaf.l2TokenAddress, leaf.chainId).decimals
     );
     leaf.refundAmounts = convertTokenListFromWei(
-      leaf.chainId,
       Array(leaf.refundAmounts.length).fill(leaf.l2TokenAddress),
       leaf.refundAmounts
     );
-    leaf.l2Token = convertTokenAddressToSymbol(leaf.chainId, leaf.l2TokenAddress);
+    leaf.l2Token = convertTokenAddressToSymbol(leaf.l2TokenAddress);
```

Comment:
`convertTokenAddressToSymbol()` seems to assume that it receives an L1 token, but we could supply an address from any chain here. Should we use `HubPoolClient.getTokenInforForAddress()` instead? We already rely on it for sourcing decimals correctly.

```suggestion
    // At line 145
    const { symbol, decimals } = hubPoolClient.getTokenInfoForAddress(leaf.l2TokenAddress, leaf.chainId);
    leaf.amountToReturn = convertFromWei(..., decimals);
 
    // ...
    
    // At line 153
    leaf.l2Token = symbol;
```